cli/tests/unit/url_test.ts

```diff
@@ -191,6 +191,21 @@ unitTest(function urlDriveLetter() {
   assertEquals(new URL("http://example.com/C:").href, "http://example.com/C:");
 });
 
+unitTest(function urlUncHostname() {
+  assertEquals(
```

Comment:
On chrome, does it also behave differently on windows vs other platforms?

That seems not right to me

Reply:
Yes, for drive letters and UNC paths.